examples/set-annotations/simple/README.md

```diff
@@ -1,35 +1,38 @@
 # set-annotations: Simple Example
 
-The `set-annotation` function adds annotations to KRM resources.
+### Overview
 
-We use the following ConfigMap to configure the function.
+This example demonstrates how to declaratively run [`set-annotations`] function
+to upsert annotations to the `.metadata.annotations` field on all resources.
+
+We use the following `Kptfile` to configure the function.
 
 ```yaml
-apiVersion: v1
-kind: ConfigMap
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
 metadata:
-  ...
-data:
-  color: orange
-  fruit: apple
+  name: example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-annotations:unstable
+      configMap:
+        color: orange
+        fruit: apple
 ```
 
-The desired annotations are provided as key-value pairs using `data` field.
+The desired annotations are provided as key-value pairs through ConfigMap.
```

Comment:
https://github.com/GoogleContainerTools/kpt/blob/next/docs/style-guides/docs.md Rule 6

Please audit all the docs for style guides